stellarphot/photometry/photometry.py

```diff
@@ -69,14 +69,8 @@ def single_image_photometry(ccd_image, sourcelist, camera, observatory_location,
         Location of the observatory where the images were taken.  Used for calculating
         the BJD.
 
-    aperture_radius : int
-        Radius of aperture(s) in pixels.
-
-    inner_annulus : int
-        Inner radius of the annulus in pixels.
-
-    outer_annulus : int
-        Outer radius of the annulus in pixels.
+    aperture_settings : `stellarphot.settings.ApertureSettings`
+        Radius, inner and outer annulus radii settings.
```

Comment:
Might be useful in the ApertureSettings documentation to include a sample call description for setting it via code.  I suppose it's like any other data class from the appearance of it, so maybe not, but just wondering.

Made a note with the ApertureSettings documentation to that effect.

Reply:
Done